src/OrbitVulkanLayer/TimerQueryPool.h

```diff
@@ -51,7 +54,9 @@ class TimerQueryPool {
       std::vector<SlotState> slots{num_timer_query_slots_};
       std::fill(slots.begin(), slots.end(), SlotState::kReadyForQueryIssue);
       device_to_query_slots_[device] = slots;
-      device_to_potential_next_free_index_[device] = 0;
+      std::vector<uint32_t> free_slots(num_timer_query_slots_);
+      std::iota(free_slots.begin(), free_slots.end(), 0);
```

Comment:
Maybe add a comment, that at the beginning, every slot is considered empty (I needed to look up the iota method).

Reply:
Done.

Reply:
Thanks!